src/main.rs

```diff
@@ -64,6 +66,11 @@ struct Opt {
         env = "CARGO_TERM_COLOR"
     )]
     color: Color,
+
+    /// Whether to enable parallel test. The `db` option will be used to create databases, and one
+    /// database will be created for each test file.
+    #[clap(long)]
```

Comment:
What about adding `-j` option?
```suggestion
    #[clap(short, long)]
```